frontend/src/components/ui/chat/ChatList.js

```diff
@@ -1,19 +1,47 @@
-import Chat from "./Chat";
-import styles from "./ChatList.module.css";
+import { useRef, useEffect } from 'react';
+
+import { v4 as uuidv4 } from 'uuid';
+
+import Chat from './Chat';
+import styles from './ChatList.module.css';
+
+const AlwaysScrollToBottom = () => {
+  const messageEndRef = useRef();
+  /*
+    false argument passed to scrollIntoView so that
+    the bottom of the element will be aligned to 
+    the bottom of the visible area of the scrollable ancestor
+  */
+  useEffect(() => messageEndRef.current.scrollIntoView(false));
+  return <div ref={messageEndRef} />;
+};
+
+function ChatList({ chats }) {
+  let lastChatMessage = chats[chats.length - 1];
```

Comment:
Should this be a `const` instead of a `let`?

Reply:
Yup thanks for pointing out, it's another misuse of `let`, my bad haha